website/docs/r/apigatewayv2_integration.html.markdown

```diff
@@ -114,7 +114,9 @@ Valid values: `AWS` (supported only for WebSocket APIs), `AWS_PROXY`, `HTTP` (su
 * `content_handling_strategy` - (Optional) How to handle response payload content type conversions. Valid values: `CONVERT_TO_BINARY`, `CONVERT_TO_TEXT`. Supported only for WebSocket APIs.
 * `credentials_arn` - (Optional) Credentials required for the integration, if any.
 * `description` - (Optional) Description of the integration.
-* `integration_method` - (Optional) Integration's HTTP method. Must be specified if `integration_type` is not `MOCK`.
+* `integration_method` - (Optional) Integration's HTTP method. Must be specified if `integration_type` is not `MOCK`. 
+For `AWS_PROXY` integrations, the method must be `POST`. For WebSocket APIs with Lambda integrations, the method must also be `POST`. 
+See [AWS CloudFormation Documentation for AWS::ApiGatewayV2::Integration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integration.html) for more details.
```

Comment:
Hey @laryssasre 👋 Thank you for taking the time to raise this to help improve our docs! I made a suggestion for a slight edit, but otherwise, this looks great to me!

- Slight rewording suggestion
- Combined into a single line
- Prefer using API documentation over CloudFormation documentation

```suggestion
* `integration_method` - (Optional) Integration's HTTP method. Must be specified if `integration_type` is not `MOCK`. For `AWS_PROXY` integrations or WebSocket APIs with Lambda integrations, the method must be `POST`. See the [AWS Documentation](https://docs.aws.amazon.com/apigatewayv2/latest/api-reference/apis-apiid-integrations.html#apis-apiid-integrationspost) for more details.
```